store-pages/logout-page/logout.css

```diff
@@ -0,0 +1,19 @@
+@import "../../common.css";
+
+.logout-flex {
```

Comment:
suggestion:- here you can extract the common properties of "logout-flex" & create different classes for flex-direction.

Reply:
I will do it later.